reflect.go

```diff
@@ -1081,6 +1088,11 @@ func (t *Schema) UnmarshalJSON(data []byte) error {
 	return json.Unmarshal(data, aux)
 }
 
+// If true, marshal json with linebreak and indent.
+var MarshalWithIndent = false
+var MarshalPrefix = ""
+var MarshalIndent = "\t"
```

Comment:
Global variables should be avoided. Wouldn't you want to do this with the regular `json.MarshalIndent` method directly?